variables.tf

```
@@ -283,6 +283,12 @@ variable "metadata_http_endpoint_enabled" {
   description = "Whether the metadata service is available"
 }
 
+variable "metadata_http_put_response_hop_limit" {
+  type        = number
+  default     = 1
```

Although 1 is more secure, it causes problems for those unaware and is hard to diagnose. Let's set the default to something more lenient and likely to work, and those in the know can set it to 1 when they are sure that it works for them.
```suggestion
  default     = 2
```